blocks/footer/footer.js

```diff
@@ -101,38 +129,35 @@ function openExternalLinksInNewTab(footer) {
   });
 }
 
-function addSocialmediaIconsToLinks(footer) {
-  footer.querySelectorAll('a[href^="https://twitter.com/"]').forEach((anchor) => {
-    anchor.innerHTML = `<i class="fa fa-twitter"></i> ${anchor.innerHTML}`;
-  });
-  footer.querySelectorAll('a[href^="https://www.facebook.com/"]').forEach((anchor) => {
-    anchor.innerHTML = `<i class="fa fa-facebook"></i> ${anchor.innerHTML}`;
-  });
-  footer.querySelectorAll('a[href^="https://www.linkedin.com/"]').forEach((anchor) => {
-    anchor.innerHTML = `<i class="fa fa-linkedin"></i> ${anchor.innerHTML}`;
-  });
-  footer.querySelectorAll('a[href^="https://www.youtube.com/"]').forEach((anchor) => {
-    anchor.innerHTML = `<i class="fa fa-youtube"></i> ${anchor.innerHTML}`;
-  });
-  footer.querySelectorAll('a[href^="https://www.instagram.com/"]').forEach((anchor) => {
-    anchor.innerHTML = `<i class="fa fa-instagram"></i> ${anchor.innerHTML}`;
-  });
+function findList(h3Element) {
+  let nextElement = h3Element.nextElementSibling;
+  while (nextElement && !nextElement.classList.contains('footer-list-item')) {
```

Comment:
`const list = h3Element.parentNode.querySelector('.footer-list-item')` would be better? 🤔